docs/source/code.rst

```diff
@@ -58,3 +58,48 @@ The purpose of the Results class is to encapsulate the results of the analysis o
 3. Code comments 
     a. Provide short description of each argument (for a function) and return value (or attribute)
     b. Specify argument / return value / attribute units
+
+Machines
+++++++++++++++++++++++++++++++++++++++++++++
+
+Machine modules should be located within ``mach_eval/machines`` and then placed within the appropriate subdirectory.
+
+Each machine module must consist of only **one** of each of the following classes:
+
+1. Machine Class
+2. Machine Operating Point Class
+
+Machine Class
+*******************************************
+
+The Machine class is where the machine definition is expected to occur. Aspects to consider:
+
+1. Required parameters
+    a. Each machine must contain parameters (such as rotor, stator, winding, etc.) defined in the ``machine`` and ``radial_machine`` classes located in the  ``mach_eval/machines`` directory.
+2. Machine characteristic dictionaries
+    a. Dictionaries must be initialized such that a ``machine`` or ``radial_machine`` is fully defined
+    b. Required dictionaries are ``dimensions``, ``parameters``, ``materials``, and ``winding``
+3. Checks
+    a. Required to validate if machine is fully defined
+    b. Check defined dictonaries against machine requirements
+    c. Throws error if machine is not fully defined
+4. Machine properties
+    a. Required for any machine variable desired in processing or post-processing
```

Comment:
These bullet points might be integrated into one section as Requirements to fully define machine above?